Test/QueryBuilderSpec.hs

```diff
@@ -192,6 +192,15 @@ tests = do
                 (toSQL theQuery) `shouldBe` ("SELECT posts.* FROM posts INNER JOIN users ON posts.created_by = users.id INNER JOIN favorite_title ON posts.title = favorite_title.title WHERE users.name != ?", [Escape "Tom"])
 
 
+        describe "labelResults" do
+            it "should provide a query with index field" do
+                let theQuery = query @Post
```

Comment:
Same as above. This test case would be simpler when just accessing the `#createdBy` field directly. I think it generally only makes sense in case you have more than one join table.